tests/functional/firefox/test_new.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.new import FirefoxNewPage
+
+
+# We don't currently show the download button for up-to-date Firefox users.
+# TODO explore how could we run this test using a known out-of-date Firefox?
+@pytest.mark.skip_if_capabilities_include({'browserName': 'firefox'})
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_click_download_button(base_url, selenium):
+    page = FirefoxNewPage(base_url, selenium, params='').open()
+    assert page.is_download_button_displayed
+    page.click_download_button()
+    assert page.is_thank_you_message_displayed
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_directly_load_thank_you(base_url, selenium):
+    page = FirefoxNewPage(base_url, selenium, params='?scene=2#download-fx').open()
```

Comment:
I'm not crazy about the `params` but don't have a better suggestion at this time. Perhaps we could subclass `FirefoxNewPage` and just alter the URL? I think my main issue is that this level of details shouldn't be in the tests if at all possible.